rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2332,6 +2337,28 @@ namespace rsx
 
 	void thread::on_notify_memory_unmapped(u32 base_address, u32 size)
 	{
+		{
+			s32 io_addr = RSXIOMem.getMappedAddress(base_address);
+			if (io_addr >= 0)
+			{
+				if (!rsx::get_current_renderer()->isHLE)
```

Comment:
copypasta. you only need (!isHLE) since this is the current renderer.

Reply:
this will look better too :)